x-pack/plugins/fleet/common/types/index.ts

```diff
@@ -43,6 +43,7 @@ export interface FleetConfigType {
     maxAgentPoliciesWithInactivityTimeout?: number;
     disableRegistryVersionCheck?: boolean;
     bundledPackageLocation?: string;
+    testSecretsIndex?: string;
```

Comment:
should this be renamed not to contain "test"?

Reply:
I have created an issue to remove this test code, I think it will be easier just to remove it at the end once everything is merged, there may be complications with the fact that the "real" index has locked down privileges so I would like to defer it for now